datacube/drivers/postgis/_spatial.py

```diff
@@ -108,6 +110,10 @@ def spindex_for_crs(crs: CRS) -> Type[SpatialIndex]:
     return spindex_for_epsg(crs.epsg)
 
 
+def normalise_crs(crs_in: CRS) -> CRS:
+    return CRS(f'epsg:{crs_in.epsg}')
+
+
 def spindex_for_record(rec: SpatialIndexRecord) -> Type[SpatialIndex]:
```

Comment:
not being used

Reply:
The intention is catch cases like:
```
crs1 = CRS('epsg:4326')
crs2 = CRS('EPSG:4326')
crs3 = CRS('WGS-84')
```
These are all the same CRS (they are functionally identical and return the same epsg code, but will fail an equality test or dictionary lookup). 

But I agree that the normalise function is not being consistently called in the right places - I will revisit.

Reply:
Might be cleaner to do it using `CRS.epsg`
(also, this particular comment was about the `spindex_for_record` method)

Reply:
Might be cleaner to do it just as `return CRS(crs_in.epsg)`
(also this particular comment was in reference to `spindex_for_record`)

Reply:
Oh I see, you're right, it's not needed any more.   It's leftover from an earlier version where the Spindex ORM registry was keyed by CRS - now that it is keyed by integer epsg number, it's no longer relevant.